test/send-test-message.js

```diff
@@ -10,17 +10,22 @@ const message = {
   'email': 'test@email.com'
 }
 
-async function sendMessage () {
-  const testConfig = { ...config.paymentQueueConfig, address: 'calculation' }
-  const messageSender = new MessageSender('test-sender', testConfig)
-  try {
-    await messageSender.openConnection()
-    const delivery = await messageSender.sendMessage(message)
-    console.log('delivered', delivery)
-  } catch (ex) {
-    console.log(ex)
```

Comment:
Connection errors were previously logged twice when running `send-test-message.js` because `messageSender.openConnection` logs errors which were then caught here. I've added similar logging to `messageSender.sendMessage` to match `openConnection` and removed this try/catch to make the script cleaner.